Sources/Benchmark/BenchmarkReporter.swift

```diff
@@ -38,28 +38,34 @@ struct PlainTextReporter<Target>: BenchmarkReporter where Target: TextOutputStre
             prefix = ""
         }
         print("running \(prefix)\(name)...", terminator: "", to: &output)
-        if output is StdoutOutputStream {
-            fflush(stdout)  // Flush stdout to actually see the message...
-        }
+        output.flush()
     }
 
     mutating func report(finishedRunning name: String, suite: String, nanosTaken: UInt64) {
         let timeDuration = String(format: "%.2f ms", Float(nanosTaken) / 1000000.0)
         print(" done! (\(timeDuration))", to: &output)
```

Comment:
Should this be `output.print` in some fashion?

Reply:
`output.print` is not aware of line terminators, so we use `print(..., to: &output)` throughout this file for consistency.

Reply:
Argh, yeah. I thought I removed my comments about that, because I figured it out later. Sorry about that, yes please ignore. :-)